packages/create-gensx/tsconfig.json

```diff
@@ -1,10 +1,29 @@
 {
-  "extends": "../tsconfig.base.json",
   "compilerOptions": {
     "tsBuildInfoFile": ".tsbuildinfo",
     "outDir": "./dist",
-    "baseUrl": "."
+    "baseUrl": ".",
+    "target": "esnext",
+    "useDefineForClassFields": true,
+    "module": "NodeNext",
+    "lib": ["ESNext"],
+    "incremental": true,
+    "skipLibCheck": true,
+    "moduleResolution": "NodeNext",
+    "resolveJsonModule": true,
+    "declaration": true,
+    "declarationMap": true,
+    "isolatedModules": true,
+    "emitDeclarationOnly": false,
+    "sourceMap": true,
+    "inlineSources": true,
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true,
+    "jsx": "react-jsx",
+    "jsxImportSource": "react"
   },
-  "include": ["src/**/*", "vitest.config.ts", "tests/**/*"],
+  "include": ["src/**/*", "tests/**/*", "vitest.config.ts"],
```

Comment:
How much of this is necessary?

Reply:
Good question. I just pasted this to match exactly what was in the `tsconfig.base.json` that we can no longer reference here (because we need jsx from react).

I could go through and clean it up but perhaps it's best just to be consistent with other packages?

Reply:
Yeah consistency is fine. Can't you still import the base config and update the JSX options?

Reply:
oh yeah that works great. updated both tsconfig files